js/src/Figure.js

```diff
@@ -80,6 +80,9 @@ var Figure = widgets.DOMWidgetView.extend({
         this.width = impl_dimensions["width"];
         this.height = impl_dimensions["height"];
 
+        this.svg.style("min-width", this.width + "px");
+        this.svg.style("min-height", this.height + "px");
```

Comment:
This will not work for the reason that I mentioned above. The `min-height` mentioned in the css will be the binding constraint and even if I increase the `min-width`, the `min-height` will determine the `el.clientHeight`.  And I cannot set the `min-height` programmatically because the `Layout` clears it out.